webrtc.html

```diff
@@ -5536,6 +5538,19 @@ <h2>Attributes</h2>
               <p>On getting, the attribute MUST return the value of the
               <a>[[\SenderRtcpTransport]]</a> slot.</p>
             </dd>
+            <dt><dfn><code>mstId</code></dfn> of type <span class=
+            "idlAttrType"><a>DOMString</a></span>, readonly, nullable</dt>
+            <dd>
+              <p>MediaStreamTrack id that appears in the msid attribute of the
+              related SDP mline or null if there is not msid attribute.
+              Useful for senders created without a track (or detached from a track).</p>
+            </dd>
+            <dt><dfn><code>kind</code></dfn> of type <span class=
+            "idlAttrType"><a>DOMString</a></span>, readonly</dt>
+            <dd>
```

Comment:
the Transceiver seems like a more natural place for kind IMO.

Reply:
Agree, since transceiver.sender.kind would have to be the same as transceiver.kind anyway.

Reply:
I agree, so I will change this property to the RtpTransceiver